pkg/cli/run.go

```diff
@@ -67,9 +74,19 @@ func Run(iacType, iacVersion string, cloudType []string,
 	outputWriter := NewOutputWriter(useColors)
 
 	if configOnly {
+		// human readable output doesn't support --config-only flag
+		// if --config-only flag is set, then display json output
+		if strings.EqualFold(format, humanOutputFormat) {
+			format = jsonOutputFormat
+		}
```

Comment:
If not supported, we may just opt to return an error and require the user to specify a valid output (-o json, yaml, etc)

Reply:
The reason the config only output is not supported is because the default scan result doesn't not hold that information. If we move the summary to ViolationStore, then we can support display the resource config with this format too. I will talk with Amir.

Reply:
done this change. Error will be displayed to user if --config-only flag is used with default human readable output.